trieste/acquisition/function.py

```diff
@@ -335,7 +335,7 @@ def probability_of_feasibility(
 
 class ExpectedConstrainedImprovement(AcquisitionFunctionBuilder):
     """
-    Builder for the _expected constrained improvement_ acquisition function defined in
+    Builder for the *expected constrained improvement* acquisition function defined in
```

Comment:
these other changes are to fix sphinx warnings, so I could debug the work easier